### ISSUE TITLE
Encoding doesnt work

### ISSUE THREAD

**Issue by schmalte04** [tidyverse/googlesheets4#121]
Currently switching from googlesheets to googlesheets4. Previoulsy i read tables through:

`gs_read(for_gs, ws = "table_name")`

Using googlesheets4 now i am using 

`sheets_find("table_name") %>% sheets_read(sheet="table_name") `

which works in RStudio fine. The mentioned table contains a lot of special characters eg. one football team name called Gençlerbirliği

Surprisingly googlesheets4 can't encode the special characters properly while running in a cronjob. It just reads Gen<U+00E7>lerbirligi SK

Did i miss to pass any encoding parameters?

**Comment by jennybc** [comment]
Duplicate of #56 #26 and fixed in https://github.com/r-lib/gargle/pull/114.

TL;DR use the dev version of gargle if you experience this problem. gargle will update on CRAN soon (has to happen before the googlesheets4 release).

**Comment by taeyounga** [comment]
hi, @jennybc , I've been trying to solve the encoding issue with  #56 #26 but i am still stuck.

What I have done:

`
remove.packages("googlesheets4")
remove.packages("gargle")
devtools::install_github("r-lib/gargle")
install.packages("googlesheets4")
`
restarted R

`
library(googlesheets4)
sheets_auth()
`

And I get following error message:
`
> sheets_auth()
Error: Can't get Google credentials.
Are you running googlesheets4 in a non-interactive session? Consider:
  * `sheets_deauth()` to prevent the attempt to get credentials.
  * Call `sheets_auth()` directly with all necessary specifics.
`

If I remove both packages again and reinstall googlesheets4, everything works again.

Below is the message I get when I install gargle dev version with 
`devtools::install_github("r-lib/gargle")`

`Enter one or more numbers, or an empty line to skip updates:
3
✓  checking for file 'C:\Users\ted\AppData\Local\Temp\Rtmpu4rl2Y\remotes24302b0824b3\r-lib-gargle-798d7d4/DESCRIPTION' (1s)
─  preparing 'gargle':
✓  checking DESCRIPTION meta-information ... 
─  checking for LF line-endings in source and make files and shell scripts
─  checking for empty or unneeded directories
─  looking to see if a 'data/datalist' file should be added
─  building 'gargle_0.4.0.9003.tar.gz'
   
* installing *source* package 'gargle' ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
  converting help for package 'gargle'
    finding HTML links ... done
    AuthState-class                         html  
    Gargle-class                            html  
Rd warning: C:/Users/ted/AppData/Local/Temp/RtmpAp4tdn/R.INSTALL50d433bb4d32/gargle/man/Gargle-class.Rd:29: file link 'Token' in package 'httr' does not exist and so has been treated as a topic
Rd warning: C:/Users/ted/AppData/Local/Temp/RtmpAp4tdn/R.INSTALL50d433bb4d32/gargle/man/Gargle-class.Rd:29: file link 'Token2.0' in package 'httr' does not exist and so has been treated as a topic
Rd warning: C:/Users/ted/AppData/Local/Temp/RtmpAp4tdn/R.INSTALL50d433bb4d32/gargle/man/Gargle-class.Rd:95: file link 'gargle_oauth_cache' in package 'gargle' does not exist and so has been treated as a topic
    GceToken                                html  
Rd warning: C:/Users/ted/AppData/Local/Temp/RtmpAp4tdn/R.INSTALL50d433bb4d32/gargle/man/GceToken.Rd:16: file link 'Token' in package 'httr' does not exist and so has been treated as a topic
Rd warning: C:/Users/ted/AppData/Local/Temp/RtmpAp4tdn/R.INSTALL50d433bb4d32/gargle/man/GceToken.Rd:16: file link 'Token2.0' in package 'httr' does not exist and so has been treated as a topic
    cred_funs                               html  
    credentials_app_default                 html  
    credentials_byo_oauth2                  html  
    credentials_gce                         html  
    credentials_service_account             html  
    credentials_user_oauth2                 html  
Rd warning: C:/Users/ted/AppData/Local/Temp/RtmpAp4tdn/R.INSTALL50d433bb4d32/gargle/man/credentials_user_oauth2.Rd:41: file link 'gargle_oauth_email' in package 'gargle' does not exist and so has been treated as a topic
Rd warning: C:/Users/ted/AppData/Local/Temp/RtmpAp4tdn/R.INSTALL50d433bb4d32/gargle/man/credentials_user_oauth2.Rd:44: file link 'gargle_oob_default' in package 'gargle' does not exist and so has been treated as a topic
Rd warning: C:/Users/ted/AppData/Local/Temp/RtmpAp4tdn/R.INSTALL50d433bb4d32/gargle/man/credentials_user_oauth2.Rd:46: file link 'gargle_oauth_cache' in package 'gargle' does not exist and so has been treated as a topic
    gargle-package                          html  
    gargle2.0_token                         html  
Rd warning: C:/Users/ted/AppData/Local/Temp/RtmpAp4tdn/R.INSTALL50d433bb4d32/gargle/man/gargle2.0_token.Rd:30: file link 'gargle_oauth_email' in package 'gargle' does not exist and so has been treated as a topic
Rd warning: C:/Users/ted/AppData/Local/Temp/RtmpAp4tdn/R.INSTALL50d433bb4d32/gargle/man/gargle2.0_token.Rd:46: file link 'gargle_oob_default' in package 'gargle' does not exist and so has been treated as a topic
Rd warning: C:/Users/ted/AppData/Local/Temp/RtmpAp4tdn/R.INSTALL50d433bb4d32/gargle/man/gargle2.0_token.Rd:52: file link 'gargle_oauth_cache' in package 'gargle' does not exist and so has been treated as a topic
    gargle_api_key                          html  
    gargle_app                              html  
    gargle_oauth_sitrep                     html  
Rd warning: C:/Users/ted/AppData/Local/Temp/RtmpAp4tdn/R.INSTALL50d433bb4d32/gargle/man/gargle_oauth_sitrep.Rd:11: file link 'gargle_oauth_cache' in package 'gargle' does not exist and so has been treated as a topic
    gargle_options                          html  
    init_AuthState                          html  
    internal-assets                         html  
    oauth_app_from_json                     html  
    request_develop                         html  
    request_make                            html  
    finding level-2 HTML links ... done

    response_process                        html  
    token-info                              html  
    token_fetch                             html  
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (gargle)`

I'm thinking the issue could be due to incomplete gargle installation. I am not sure what i should do in this case :(
These "file link 'gargle_oauth_cache' in package 'gargle' does not exist and so has been treated as a topic" messages seems to be the problem.

Thanks for your help!

Taeyoung

**Comment by jennybc** [comment]
These Rd warnings are actually quite harmless and are often seen on Windows.

If you actually have a usage problem at this point, please open a new issue. I can't tell -- it sounds like maybe things are working now?

**Comment by msrodrigues** [comment]
![image](https://user-images.githubusercontent.com/7436194/79758123-5d2b3d00-82f3-11ea-91c2-3fd039f7918c.png)
When I call the DF it shows the wrong encoding, even though when I call the specific column it appears to work just fine. :/

**Comment by jennybc** [comment]
@msrodrigues Is that a public Sheet? For us to get somewhere, we need an example that I (and possibly the maintainer of tibble/pillar) needs to be able to run. If you can get a proper reprex, please open it as a new issue.

Just to confirm: you're using dev googlesheets4 and dev gargle, yes? You should be.

https://googlesheets4.tidyverse.org/articles/articles/googlesheets4-reprex.html

**Comment by msrodrigues** [comment]
CRAN googlesheets4 and DEV gargle. I will try to change googlesheets4 for dev version and check again. If still not working I will open a new issue. Thanx
